tritium-caffeine/src/main/metrics/cache-metrics.yml

```diff
@@ -0,0 +1,38 @@
+options:
+  javaPackage: com.palantir.tritium.metrics.caffeine
+  javaVisibility: packagePrivate
+namespaces:
+  cache:
+    docs: Cache statistic metrics
+    metrics:
+      hit.count:
+        type: counter
+        tags: [cache]
+        docs: Count of cache hits
+      miss.count:
+        type: counter
+        tags: [cache]
+        docs: Count of cache misses
+      load.success.count:
+        type: counter
+        tags: [cache]
+        docs: Count of successful cache loads
+      load.failure.count:
+        type: counter
+        tags: [cache]
+        docs: Count of failed cache loads
```

Comment:
Thinking out loud here, entirely up to you either way:

We could declare a single metric along these lines, using a tag to distinguish success and failure:
```yaml
load:
  type: meter
  tags:
    - name: cache
    - name: result
       values: [success, failure]
```

Sometimes this makes it easier to track total interactions